frontend/src/metabase/visualizations/echarts/cartesian/model/other-series.ts

```diff
@@ -100,55 +100,75 @@ const AGGREGATION_FN_MAP: Record<
   AggregationType,
   { fn: (values: number[]) => number; label: string }
 > = {
-  count: {
-    label: t`Total`,
-    fn: sum,
+  get count() {
+    return {
+      label: t`Total`,
+      fn: sum,
+    };
   },
-  sum: {
-    label: t`Total`,
-    fn: sum,
+  get sum() {
+    return {
+      label: t`Total`,
+      fn: sum,
+    };
   },
-  "cum-sum": {
-    label: t`Total`,
-    fn: sum,
+  get "cum-sum"() {
+    return {
+      label: t`Total`,
+      fn: sum,
+    };
   },
-  "cum-count": {
-    label: t`Total`,
-    fn: sum,
+  get "cum-count"() {
+    return {
+      label: t`Total`,
+      fn: sum,
+    };
   },
-  avg: {
-    label: t`Average`,
-    fn: values => sum(values) / values.length,
+  get avg() {
+    return {
+      label: t`Average`,
+      fn: (values: number[]) => sum(values) / values.length,
```

Comment:
it seems the type in `fn` can't be inferred from the `Record`'s value type. But it checks the type, so if you return something else in each getter, TS would complain.